File: Data-Types/1. Character/2. Identification/Character Identification.java
import java.util.Scanner;

class Identifier
{
    public static void main(String args[])
    {
        Scanner s=new Scanner(System.in);
        
        char ch;
        System.out.print("\n Enter a character : ");
        ch=s.next().charAt(0);
        
        if(ch>=48 && ch<=57)
         System.out.print("\n '"+ch+"' is a numeral digit. \n");
        else if(ch>=65 && ch<=90)
         System.out.print("\n '"+ch+"' is an uppercase alphabet. \n");
        else if(ch>=97 && ch<=122)
         System.out.print("\n '"+ch+"' is a lowercase alphabet. \n");
        else
         System.out.print("\n '"+ch+"' is a special character. \n");
    }
}

File: Data-Types/1. Character/2. Identification/Character Identification.cs
using System;

namespace ASCII
{
    class ASCII
    {
        static void Main(string[] args)
        {
            char ch;
            Console.Write("\n Enter a character : ");
            ch = char.Parse(Console.ReadLine());

            int n=ch;
            if (n >= 48 && n <= 57)
                Console.Write("\n '" + ch + "' is a numeral digit. \n");
            else if (n >= 65 && n <= 90)
                Console.Write("\n '" + ch + "' is an uppercase alphabet. \n");
            else if (n >= 97 && n <= 122)
                Console.Write("\n '" + ch + "' is a lowercase alphabet. \n");
            else
                Console.Write("\n '" + ch + "' is a special character. \n");
            Console.ReadKey();
        }
    }
}